src/components/step/ExampleStep/ExampleStep.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import '../step.css';
+
+export interface ExampleStepProps {
+    children?: React.ReactNode;
+}
+
+export default function (props: ExampleStepProps) {
```

Comment:
So I think the main goal here seems to be to include the main CSS file for storybook purposes?

I would instead follow the pattern with the other components in this library where I actually have a ```<Layout>``` component that embeds the "global" CSS stuff.  Here I might call it ```<StepLayout>```

Then your YoutubeStep would actually have ```<StepLayout>``` as it's top level dev (and all future steps would) and we don't have a component that's used only for storybook purposes